src/main/java/edu/iis/powp/events/predefine/SelectComplexCommandOptionListener.java

```diff
@@ -16,14 +22,20 @@ public SelectComplexCommandOptionListener(int figureType)
     @Override
     public void actionPerformed(ActionEvent e)
     {
-
+        ComplexCommand cc;
         switch(figureType)
         {
             case 1:
-                System.out.println("Figura 1.");
+                cc = FigureFactory.square();
+                cc.execute(Application.getComponent(DriverManager.class).getCurrentPlotter());
+                break;
+            case 2:
+                cc = FigureFactory.star();
+                cc.execute(Application.getComponent(DriverManager.class).getCurrentPlotter());
                 break;
             default:
-                System.out.println("Figura 2.");
+                cc = FigureFactory.triangle();
+                cc.execute(Application.getComponent(DriverManager.class).getCurrentPlotter());
```

Comment:
Application.getComponent(DriverManager.class).getCurrentPlotter() duplicated 2 times in the method.